src/util/helpers.tsx

```diff
@@ -159,8 +159,4 @@ export const getUrlParam = (paramName: string, url?: string): string | null => {
 export const defaultFirst = (p1: { name: string }, p2: { name: string }) =>
   p1.name === "default" ? -1 : p2.name === "default" ? 1 : 0;
 
-export const isWidthBelow = (width: number) => {
-  // using the max from both, because there is a bug in chrome, causing a 0 outerWidth for
-  // background tabs: https://bugs.chromium.org/p/chromium/issues/detail?id=719296
-  return Math.max(window.outerWidth, window.innerWidth) < width;
-};
+export const isWidthBelow = (width: number) => window.innerWidth < width;
```

Comment:
This was creating a much bigger problem than the small one it was solving. Taking the max of inner and outer width is not a good approach. We're not getting the breakpoint right with this approach, because what we need here is just the `innerWidth`. We don't really care about the outer one. When we're not opening a tab in the background with a small width (which is 99.9% of the use case, I'd say), we are taking the `outerWidth` with this approach, which is consistently bigger than the `innerWidth` (which is the correct one instead, and the one that we should always use). So when our breakpoint is 620, with the "max approach" we'd need to pass in 692 to the function in order to get that breakpoint right. But this will break other browsers (Firefox) because they don't have this bug. You'd need to pass just 620 for them, not 692. Hence my correction. If you have a better idea, that solves the small (negligible) problem that you mentioned without creating the much bigger one that I just described, feel free to propose it, but this logic of taking the max is broken and doesn't make much sense, as @aaryanporwal was also trying to tell us the other day.

Reply:
The `innerWidth` is not affected by the bug and actually the right thing to use, because it also encounters for zoom factor (which might explain the differences you encountered). I don't remember why we back in the day started with `outherWidth` and ran into the chrome bug.

So yeah, let's simplify this as proposed.